FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -137,8 +161,18 @@ await foreach (FhirToDataLakeProcessingJobInputData input in inputs.WithCancella
 
                     _result.SequenceIdToJobIdMapForRunningJobs[input.ProcessingJobSequenceId] = newJobId;
 
-                    // if enqueue successfully while fails to report result, will re-enqueue and return the existing jobInfo
-                    progress.Report(JsonConvert.SerializeObject(_result));
+                    if (input.SplitParameters != null)
```

Comment:
in what scenarios SplitParameters is null?

Reply:
System scope will always not null.